### PR TITLE
Branch ugdg

### PR DESCRIPTION
**Pull request by chaosInTheWild**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ with the specified index.
 The following sequence diagram below shows how the delete feature works. Note the <code>Ui</code> class is
 omitted in the sequence diagram to emphasise on the other classes:
 
-![alt text](images/Delete.png)
+![alt text](images/DeleteFeature.png)
```
Comment (trishaangelica) [inline docs/DeveloperGuide.md:30]: nice name change! :)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,33 +1,59 @@
-# User Guide
+# SHOCO v1.0 - User Guide
 
 ## Introduction
 
-{Give a product intro}
+SHOCO is a command-line interface (CLI) application that allows users to 
+manage and plan their shopping lists and budget. If you are a fast typer, 
+you will find SHOCO to be even more effective than applications with graphical 
+user interfaces.
```
Comment (trishaangelica) [inline docs/UserGuide.md:8]: thanks for adding in the introduction for the team!

**Review by trishaangelica — COMMENTED** [review]
LGTM!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (chaosInTheWild) [reply docs/DeveloperGuide.md:30]: Thanks!

File: docs/UserGuide.md (hunk 2 above)
Comment (chaosInTheWild) [reply docs/UserGuide.md:8]: No worries!

**Pull request merged**